lib/honeykiq/server_middleware.rb

```diff
@@ -25,16 +25,28 @@ def extra_fields
 
     private
 
-    def call_event(event, msg, queue_name)
-      event.add(default_fields(msg, queue_name))
-      duration_ms(event) { yield }
+    def start_span(name:)
+      if @honey_client
```

Comment:
maybe would be nice to name this
not sure what a good name would be, maybe `using_libhoney?` or just `libhoney?`